node/task.ts

```diff
@@ -1531,6 +1531,41 @@ export function findMatch(defaultRoot: string, patterns: string[] | string, find
 }
 
 //-----------------------------------------------------
+// Http Proxy Helper
+//-----------------------------------------------------
+
+export interface ProxyConfiguration {
+    proxyUrl: string;
+    proxyUsername?: string;
```

Comment:
should this be `proxyUserName` since user name are two words? or should this interface match some other convention (e.g. match an interface exposed by node?)